src/test/resources/com/tresys/nitf/xsd/nitf_ude_types.dfdl.xsd

```diff
@@ -0,0 +1,156 @@
+
+<xs:schema targetNamespace="urn:nitfUserDefinedExtensions" xmlns:tns="urn:nitfUserDefinedExtensions"
+  xmlns:xs="http://www.w3.org/2001/XMLSchema"
+  xmlns:fn="http://www.w3.org/2005/xpath-functions"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns:dfdl="http://www.ogf.org/dfdl/dfdl-1.0/"
+  xmlns:dfdlx="http://www.ogf.org/dfdl/dfdl-1.0/extensions"
+  xmlns:nitf="urn:nitf:2.1"
+  elementFormDefault="unqualified">
+
+  <xs:import namespace="urn:nitf:2.1" schemaLocation="com/tresys/nitf/xsd/nitf.dfdl.xsd" />
+
+  <xs:import schemaLocation="com/mitre/jpeg/xsd/jpeg.dfdl.xsd" />
+
+  <xs:annotation>
+    <xs:appinfo source="http://www.ogf.org/dfdl/">
+      <dfdl:format representation="text" alignment="1" alignmentUnits="bytes"
+        binaryBooleanFalseRep="0" binaryBooleanTrueRep="1"
+        binaryCalendarEpoch="1970-01-01T00:00:00+00:00" binaryCalendarRep="bcd"
+        binaryDecimalVirtualPoint="0" binaryFloatRep="ieee"
+        binaryNumberCheckPolicy="lax" binaryNumberRep="binary"
+        binaryPackedSignCodes="C D F C" calendarCenturyStart="53"
+        calendarCheckPolicy="lax" calendarDaysInFirstWeek="4"
+        calendarFirstDayOfWeek="Monday" calendarLanguage="en-US"
+        calendarObserveDST="yes" calendarPatternKind="implicit"
+        calendarPattern="yyyy-MM-dd'T'HH:mm:ss" calendarTimeZone="UTC"
+        choiceLengthKind="implicit" decimalSigned="yes"
+        documentFinalTerminatorCanBeMissing="no" emptyValueDelimiterPolicy="none"
+        encoding="us-ascii" encodingErrorPolicy="replace" escapeSchemeRef=""
+        fillByte="%NUL;" floating="no" ignoreCase="no" initiatedContent="no"
+        initiator="" leadingSkip="0" lengthKind="implicit" lengthUnits="bytes"
+        occursCountKind="implicit" outputNewLine="%CR;%LF;"
+        prefixIncludesPrefixLength="no" separator=""
+        separatorPosition="infix" sequenceKind="ordered" terminator=""
+        textBidi="no" textBooleanFalseRep="false" textBooleanJustification="left"
+        textBooleanPadCharacter="%SP;" textBooleanTrueRep="true"
+        textCalendarJustification="left" textCalendarPadCharacter="%SP;"
+        textNumberCheckPolicy="strict" textNumberJustification="right"
+        textNumberPadCharacter="0" textNumberPattern="#0" textNumberRep="standard"
+        textNumberRoundingIncrement="0.0" textNumberRoundingMode="roundUp"
+        textNumberRounding="pattern" textOutputMinLength="0" textPadKind="padChar"
+        textStandardBase="10" textStandardDecimalSeparator="."
+        textStandardGroupingSeparator="," textStandardInfinityRep="Inf"
+        textStandardNaNRep="NaN" textStandardZeroRep="" textStandardExponentRep=""
+        textStringJustification="left" textStringPadCharacter="%SP;"
+        textTrimKind="none" textZonedSignStyle="asciiStandard" trailingSkip="0"
+        truncateSpecifiedLengthString="no" utf16Width="fixed"
+        bitOrder="mostSignificantBitFirst" byteOrder="bigEndian"/>
+    </xs:appinfo>
+  </xs:annotation>
```

Comment:
If we do the nitf-core-types.dfdl.xsd thing, we might also want to define a format that extensions can use so they don't need to copy paste this whole blob.